CHANGELOG.md

```diff
@@ -7,13 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
-- None.
+- **fixed:** Fix `extract::Path` not percent decoding params ([#272](https://github.com/tokio-rs/axum/pull/272))
```

Comment:
This should be `**changed**` and the PR labeled 'breaking change'. Doing more or less percent encoding clearly changes observable behavior.

Reply:
Bah you're right. One more thing that has to wait for 0.3 😞